styles/utils/mixins.scss

```diff
@@ -0,0 +1,38 @@
+@mixin for-mobile-big() {
+  @media (min-width: 744px) {
```

Comment:
The media query for mobile devices seems to have a min-width of 744px. This can cause some styling issues on smaller screens. Consider adjusting this value to cater to smaller screens, typically 480px is used for mobile devices.